applications/StructuralMechanicsApplication/custom_constitutive/linear_plane_stress.cpp

```diff
@@ -117,9 +116,8 @@ void LinearPlaneStress::CalculatePK2Stress(
     ConstitutiveLaw::Parameters& rValues
 )
 {
-    const Properties& r_material_properties = rValues.GetMaterialProperties();
-    const double E = r_material_properties[YOUNG_MODULUS];
-    const double NU = r_material_properties[POISSON_RATIO];
+    const double E  = GetMaterialProperty(YOUNG_MODULUS, rValues);
+    const double NU = GetMaterialProperty(POISSON_RATIO, rValues);
```

Comment:
what about the elements?
=> what if they use `r_material_properties[YOUNG_MODULUS]`?

Reply:
hmmm should they? I'm thinking about standard solid FE. In what cases this can happen?

Reply:
I am not sure
a quick search showed that it is not used often but sometimes
Can you take a look yourself at those cases?